test/fixtures/connect_add_trans.json

```diff
@@ -0,0 +1,511 @@
+      },
+      "category": [
+        "Transfer",
+        "Third Party",
+        "Venmo"
+      ],
+      "category_id": "21010001",
+      "score": {
+        "location": {},
+        "name": 1
+      }
+    },
+    {
+      "_account": "nban4wnPKEtnmEpaKzbYFYQvA7D7pnCaeDBMy",
+      "_id": "moPE4dE1yMHJX5pmRzwrcvpkxdopLxtEKPREYo",
+      "amount": 120,
+      "date": "2014-04-19",
+      "name": "Venmo Payment 16991172",
+      "meta": {
+        "location": {}
+      },
+      "pending": false,
+      "type": {
+        "primary": "special"
+      },
+      "category": [
+        "Transfer",
+        "Third Party",
+        "Venmo"
+      ],
+      "category_id": "21010001",
+      "score": {
+        "location": {},
+        "name": 1
+      }
+    },
+    {
+      "_account": "pJPM4LMBNQFrOwp0jqEyTwyxJQrQbgU6kq37k",
+      "_id": "JmN0JX0q5EcaQJM9ZbOwUYyyp607m4u3PR63Vn",
+      "amount": 5.32,
+      "date": "2014-04-17",
+      "name": "Octane Coffee Bar and Lounge",
+      "meta": {
+        "location": {
+          "address": "1009 Marietta St Nw # B",
+          "city": "Atlanta",
+          "state": "GA"
+        }
+      },
+      "pending": false,
+      "type": {
+        "primary": "place"
+      },
+      "category": [
+        "Food and Drink",
+        "Restaurants",
+        "Coffee Shop"
+      ],
+      "category_id": "13005043",
+      "score": {
+        "location": {
+          "address": 1,
+          "city": 1,
+          "state": 1
+        },
+        "name": 0.2
+      }
+    },
+    {
+      "_account": "pJPM4LMBNQFrOwp0jqEyTwyxJQrQbgU6kq37k",
+      "_id": "4r0aBVa85Kt3BDPk10a4U5OD3XKjE7Hzxpez6B",
+      "amount": 28.57,
+      "date": "2014-04-11",
+      "name": "Papa Johns Pizza",
+      "meta": {
+        "location": {
+          "address": "2625 David Dr",
+          "city": "Metairie",
+          "coordinates": {
+            "lat": 29.999986,
+            "lon": -90.21869
+          },
+          "state": "LA",
+          "zip": "70003"
+        }
+      },
+      "pending": false,
+      "type": {
+        "primary": "place"
+      },
+      "category": [
+        "Food and Drink",
+        "Restaurants",
+        "Pizza"
+      ],
+      "category_id": "13005012",
+      "score": {
+        "location": {
+          "address": 1,
+          "city": 1,
+          "state": 1,
+          "zip": 1
+        },
+        "name": 0.2
+      }
+    },
+    {
+      "_account": "QPO8Jo8vdDHMepg41PBwckXm4KdK1yUdmXOwK",
+      "_id": "eJXpMzpR65FP4RYno6yjhBDxgkM6pXC9RKM9o3",
+      "amount": -3042.44,
+      "date": "2014-03-27",
+      "name": "Company Payroll",
+      "meta": {
+        "location": {}
+      },
+      "pending": false,
+      "type": {
+        "primary": "special"
+      },
+      "category": [
+        "Transfer",
+        "Payroll"
+      ],
+      "category_id": "21009000",
+      "score": {
+        "location": {},
+        "name": 1
+      }
+    }
+  ],
+  "access_token": "test_wells"
+}
```

Comment:
if you have a moment could you set your editor to automatically insert newlines at end of file?